sky/execution.py

```diff
@@ -267,6 +267,12 @@ def _execute(
                     # no-credential machine should not enter optimize(), which
                     # would directly error out ('No cloud is enabled...').  Fix
                     # by moving `sky check` checks out of optimize()?
+
+                    controller = controller_utils.Controllers.from_name(
+                        cluster_name)
+                    if controller is not None:
+                        logger.info(
+                            f'Choosing resources for {controller.name}...')
```

Comment:
What's the rationale behind this logging? I found it a bit odd to see.

Reply:
I guess this is for the case when the `sky jobs launch` is called and there can be two optimizer tables: one for the job itself, and another is the table for the controller.

I think it is good to clarify that the second optimizer table is for controller, but it would be nice to elaborate this, e.g., changing some of our logging to be `Launching managed jobs controller: {controller.name}`

Reply:
How about "Resources for jobs/serve controller:"? (Lower case and no underscore)

Reply:
Ahhh that's a bug. Taking a look

Reply:
#4376